docker/daemon/daemon_transport_test.go

```diff
@@ -51,8 +51,8 @@ func testParseReference(t *testing.T, fn func(string) (types.ImageReference, err
 		{"sha256:XX23456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef", "", ""}, // Invalid digest value
 		{"UPPERCASEISINVALID", "", ""},                                                      // Invalid reference input
 		{"busybox", "", ""},                                                                 // Missing tag or digest
-		{"busybox:latest", "", "docker.io/library/busybox:latest"},                          // Explicit tag
-		{"busybox@" + sha256digest, "", "docker.io/library/busybox@" + sha256digest},        // Explicit digest
+		{"busybox:latest", "", "<local>/library/busybox:latest"},                            // Explicit tag
+		{"busybox@" + sha256digest, "", "<local>/library/busybox@" + sha256digest},          // Explicit digest
```

Comment:
See the long comment in `docker/tarfile/Destination.PutManifest`, this would break interoperability with projectatomic/docker.